build_vercel.py

```
@@ -0,0 +1,32 @@
+
+def run():
+    print("--- Vercel Build Script ---")
+    
+    # 1. Attempt to compile C++ engine
+    cpp_source = "game/engine/main.cpp"
+    cpp_output = "game/engine/main"
+    
+    print(f"Attempting to compile {cpp_source}...")
+    try:
+        # We try to use g++ if available
+        subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)